dashboard/src/components/AppView/AppSecrets/AppSecrets.scss

```diff
@@ -5,5 +5,4 @@
 
 .app-secrets-section {
   max-height: 10rem;
-  overflow-y: auto;
```

Comment:
this fixes https://github.com/kubeapps/kubeapps/issues/2462 ?

Reply:
Perhaps, I am missing something, but as far as I have checked, I would say so

![secretsScroll](https://user-images.githubusercontent.com/11535726/109788351-85701180-7c0f-11eb-8570-a0af9f76ad08.gif)

Reply:
great, thanks for checking!